research/picai/fedavg/server.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+from functools import partial
+from logging import INFO
+from typing import Any, Dict
+
+import flwr as fl
+from flwr.common.logger import log
+from flwr.server.client_manager import SimpleClientManager
+from flwr.server.strategy import FedAvg
+
+from fl4health.parameter_exchange.full_exchanger import FullParameterExchanger
+from fl4health.utils.config import load_config
+from research.picai.fl_utils import (
+    evaluate_metrics_aggregation_fn,
+    fit_config,
+    fit_metrics_aggregation_fn,
```

Comment:
Can we instead directly use `examples.simple_metric_aggregation` for `evaluate_metrics_aggregation_fn` and `fit_metrics_aggregation_fn`?
And instead of  `get_initial_model_parameters` can we use `fl4health.utils.functions import get_all_model_parameters`? In that case, we probably would not need  to have`fl_utils.py`